cmd/commands/use.go

```diff
@@ -87,6 +88,12 @@ func useCmd(ctx *cli.Context) error {
 	if err != nil {
 		return fmt.Errorf("%s not supported, error: %w", name, err)
 	}
+
+	version, err = source.PreUse(version, scope)
```

Comment:
Put this code into Sdk#Use, there is no need to implement a separate SDK#PreUse function.

Reply:
Regarding this scenario, I've thought about it a bit, and I don't think it's a good experience to be compatible with other config file via `vfox use` command.

As I say https://github.com/version-fox/vfox/issues/56#issuecomment-1970743723, the use command simply writes the version information to `.tool-versions` file. What actually works is the env command.

So now I'm thinking, maybe implementing a `PreEnv` func or modifying `EnvKeys` func might be a better solution for compatibility with other config file.

Thought?

Reply:
In other words, the best case scenario for compatibility is that `vfox` can read `.nvmrc` directly without use `use` command when the user enters the project.